src/component/list.js

```diff
@@ -50,7 +50,7 @@ export class List extends Component {
         ref={component => (this.list = component)}
         dataSource={this.dataSource}
         renderHeader={this.props.renderHeader}
-        stickyHeaderIndices={[0]}
+        stickyHeaderIndices={this.props.renderHeader ? [0] : []}
```

Comment:
Why is this needed?

Reply:
Otherwise, if you have a list with no header then the first item of the list sticks to the top of the page on scroll.